bin/rigger-save-vars

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+export VARIABLES
+
+function usage {
+  cat <<EOF
+${BASH_SOURCE[0]} <variable name>
```

Comment:
Probably just habit (using BASH_SOURCE) because I write a lot of bash functions where $0 would actually give the name of the overarching script, not the sourced script. In this case, `$0` would work just as well.